scripts/test_server.py

```
@@ -0,0 +1,301 @@
+        for route in list(self.app.router.routes()):
+            cors.add(route)
+
+    async def serve_dashboard(self, request):
+        """Serve the test dashboard HTML"""
+        web_dir = Path(__file__).parent.parent / "web"
+        dashboard_path = web_dir / "test-dashboard.html"
+
+        if dashboard_path.exists():
+            return web.FileResponse(dashboard_path)
+        else:
+            return web.Response(text="Dashboard not found", status=404)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)